src/prop/cnf_stream.cpp

```diff
@@ -363,21 +355,18 @@ SatLiteral CnfStream::handleAnd(TNode andNode)
   // Number of children
   unsigned n_children = andNode.getNumChildren();
 
-  // Transform all the children first (remembering the negation)
-  TNode::const_iterator node_it = andNode.begin();
-  TNode::const_iterator node_it_end = andNode.end();
-  SatClause clause(n_children + 1);
-  for(int i = 0; node_it != node_it_end; ++node_it, ++i) {
-    clause[i] = ~toCNF(*node_it);
-  }
-
   // Get the literal for this node
   SatLiteral andLit = newLiteral(andNode);
 
-  // lit -> (a_1 & a_2 & a_3 & ... & a_n)
-  // ~lit | (a_1 & a_2 & a_3 & ... & a_n)
-  // (~lit | a_1) & (~lit | a_2) & ... & (~lit | a_n)
-  for(unsigned i = 0; i < n_children; ++i) {
+  // Transform all the children first (remembering the negation)
+  SatClause clause(n_children + 1);
```

Comment:
Since we are changing this code anyway, could we change the type of `n_children` to `size_t` and maybe change the name to camelCase (same for `n_children` above)?